src/scribe_data/cli/get.py

```diff
@@ -172,52 +167,55 @@ def prompt_user_download_all():
         print(
             f"Updating data for language(s): {language.title()}; data type(s): {data_type.capitalize()}"
         )
-        query_data(
+        existing_files = list(Path(output_dir).glob(f"{language}/{data_type}.json"))
+        if existing_files:
+            print(
+                f"Existing file(s) found for {language.title()} and {data_type.capitalize()} in the {output_dir} directory."
+            )
+            for idx, file in enumerate(existing_files, start=1):
+                print(f"{idx}. {file.name}")
+
+            print("\nChoose an option:")
+            print("1. Overwrite existing data (press 'o')")
+            print("2. Skip process (press anything else)")
+            user_choice = input("Enter your choice: ").strip().lower()
+
+            if user_choice == "o":
+                print("Overwrite chosen. Removing existing files...")
+                for file in existing_files:
+                    file.unlink()
+            else:
+                print(f"Skipping update for {language.title()} {data_type}.")
+                return {"success": False, "skipped": True}
+
+        query_result = query_data(
```

Comment:
Here, as you can see [existing files check](https://github.com/scribe-org/Scribe-Data/blob/dd50d8dcbdbd624e84da9d5897f1ecd6c8adb88d/src/scribe_data/wikidata/query_data.py#L186) already done in [query_result](https://github.com/scribe-org/Scribe-Data/blob/dd50d8dcbdbd624e84da9d5897f1ecd6c8adb88d/src/scribe_data/wikidata/query_data.py#L99) function. Can we reconsider by ignoring the check before calling the query_result method?